coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -210,6 +250,7 @@ export default {
 
     loadData () {
       this.taskItems = []
+      this.confirmCheckbox = this.entityFilter(EntityTypes.COOP)
```

Comment:
I am unclear about the business rules around this statement. Please add a high level comment, something like "is not used for Coops so set this to true to make page logic work".
FWIW, although this logic is correct, it would be more clear to code "in the case of Coop, set this to true" (otherwise it stays initialized to false as per data() ).

Reply:
I can definitely write this a little more clearly, at first glance it doesn't give you much information of the actual purpose of this line of code.

Reply:
Thanks for the change. I think this will help anyone else looking at that code.